app/views/components/blockEditor.html

```diff
@@ -204,19 +204,86 @@
                 <span class="help-block help-block-hidden" ng-if="ruleValues(rule.parentBlockId).min && !ruleValues(rule.parentBlockId).max" translate>Please enter a number greater than or equal to {{ruleValues(rule.parentBlockId).min}}</span>
                 <span class="help-block help-block-hidden" ng-if="!ruleValues(rule.parentBlockId).min && ruleValues(rule.parentBlockId).max" translate>Please enter a number less than or equal to {{ruleValues(rule.parentBlockId).max}}</span>
               </div>
-              <select ng-switch-when="select" ng-model="rule.value" ng-options="v for v in ruleValues(rule.parentBlockId)" class="form-control"></select>
-              <select ng-switch-when="gender" ng-model="rule.value" class="form-control">
+              <select ng-switch-when="select" show-errors ng-model="rule.value" ng-options="v for v in ruleValues(rule.parentBlockId)" class="form-control"></select>
+              <select ng-switch-when="gender" show-errors ng-model="rule.value" class="form-control">
                 <option value="M" translate>Male</option>
                 <option value="F" translate>Female</option>
               </select>
-              <pick-a-date ng-switch-when="date" ng-model="rule.value" picker-min-date="getRangeValues(rule.parentBlockId).min" picker-max-date="getRangeValues(rule.parentBlockId).max"></pick-a-date>
+              <pick-a-date ng-switch-when="date" show-errors ng-model="rule.value" picker-min-date="getRangeValues(rule.parentBlockId).min" picker-max-date="getRangeValues(rule.parentBlockId).max"></pick-a-date>
             </div>
             <div class="clearfix visible-sm-block"></div>
             <div class="col-md-1 text-right">
               <button class="btn btn-danger" ng-click="removeRule(rule.id)" title="Delete Rule"><i class="fa fa-trash-o"></i></button>
             </div>
           </div>
         </div>
+
+        <div ng-if="block.type === 'checkboxQuestion' && disableForceSelectionRule() === false">
```

Comment:
@OzzieOrca The new rules section is hidden by default. It becomes visible beneath the existing rules section only when the administrator does the following:

1. Check an option, which brings up a new check box "Force Selection" to the right.
2. Check the "Force Selection" check box.

  You may have missed it, since it is visible only if the edit panel is open.